sc/src/main/javascript/webpack.config.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
     atlasAutocomplete: `./bundles/autocomplete`,
     experimentPage: `./bundles/experiment-page`,
     geneSearch: `./bundles/gene-search`,
-    geneSearchForm: `./bundles/gene-search-form`
+    geneSearchForm: `./bundles/gene-search-form`,
+    atlasHomepageCard: `./bundles/atlas-homepage-cards`
```

Comment:
I see that you added an entry to the Webpack config and in the bundles, but the directory `bundles/atlas-homepage-cards` is missing. Could you add it too?